Execute Async Fn in paralle/code.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * @param {Array<Function>} functions
+ * @return {Promise<any>}
+ */
+var promiseAll = function (functions) {
+    return new Promise((resolve, reject) => {
+        const res = new Array(functions.length);
```

Comment:
```suggestion
        const result = [];
```
no need for generating same length array here.